packages/mdc-top-app-bar/constants.js

```diff
@@ -17,9 +17,17 @@
 /** @enum {string} */
 const strings = {
   NAVIGATION_EVENT: 'MDCTopAppBar:nav',
+  TOPAPPBAR_SELECTOR: '.mdc-top-app-bar',
```

Comment:
Rename to `ROOT_SELECTOR` (we have `ROOT` in other constants files for the top-level class)